### PR TITLE
[MNGSITE-539] Add Maven CI Friendly Versions to menu

### PR DESCRIPTION
**Pull request by Bukama**
The guide about Maven CI Friend Versions was integrated into the navigation menu. As it is a guide, the file was moved to the guide section and renamed to match the naming schema of this folder. Therefor the release notes of Maven 3.5.0 (and its beta) was updated to the new links. Also some small formatting errors in those release notes were fixed in the same action.

### CODE REVIEW

**Comment by Bukama** [comment]
Note: Spotless was applied and IDE is set to the [Maven code styles](https://maven.apache.org/developers/conventions/code.html#Generic_Code_Style_And_Convention) (for MD files esp. 120 chars per line). The files were not matching the code styles before (lines of 177 chars and more)

**Comment by Bukama** [comment]
> Can you split the PR so that we can review the reformat / actual changes ?

Yeah sure. Opened #595 to just contain the formatting of the effected files. Will then rebase/redo the actual changes to add the guide to the menu etc.

**Comment by slawekjaranowski** [comment]
#595 merged

**Comment by Bukama** [comment]
> #595 merged

Thank you, rebased this branch.

File: content/markdown/docs/3.5.0-beta-1/release-notes.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ you are expecting to build your project with, instead of using `prerequisites`
   has been fixed [MNG-6057], [MNG-6090] and [MNG-5895]. It is very important to
   know if you are using the previously named properties for a version in your
   pom you have to use [flatten-maven-plugin] if you like to do an `mvn install`
-  or `mvn deploy` more details can be found at [Maven CI Friendly](/maven-ci-friendly.html).
+  or `mvn deploy` more details can be found at [Maven CI Friendly Versions](/guides/mini/guide-maven-ci-friendly.html).
```
Comment (elharo) [inline content/markdown/docs/3.5.0-beta-1/release-notes.md:149]: run-on sentence

File: content/markdown/docs/3.5.0-beta-1/release-notes.md [hunk 2]
```diff
@@ -160,9 +160,8 @@ Bugs:
 * [MNG-6170] - NPE in cases using Multithreaded -T X versions:set -DnewVersion=1.0-SNAPSHOT
 * [MNG-6173] - MavenSession.getAllProjects() should return all projects in the reactor
 * [MNG-6176] - Javadoc errors prevent release with Java 8
-* [MNG-6177] - The `--file` command line option of the Windows and Unix launchers does not work for directory names
-  like "
-  Spaces & Special Char"
+* [MNG-6177] - The `--file` command line option of the Windows and Unix launchers does not work for directory names like
```
Comment (elharo) [inline content/markdown/docs/3.5.0-beta-1/release-notes.md:163]: but 9it does work on Macs?

**Review by elharo — CHANGES_REQUESTED** [review]

File: content/markdown/docs/3.5.0-beta-1/release-notes.md (hunk 1 above)
Comment (Bukama) [reply content/markdown/docs/3.5.0-beta-1/release-notes.md:149]: > run-on sentence

Can you explain what you mean? Do you want me to change the "to do an `mvn install`" to "to run `mvn install`"? 
My PR does not aim to change the wording of the release notes, but I'm open to do it.

File: content/markdown/docs/3.5.0-beta-1/release-notes.md (hunk 2 above)
Comment (Bukama) [reply content/markdown/docs/3.5.0-beta-1/release-notes.md:163]: I don't know, maybe @cstamas @gnodet @slawekjaranowski do? To be fair the ticket was closed in 2017

Comment (elharo) [reply content/markdown/docs/3.5.0-beta-1/release-notes.md:163]: I'm not clear on why you're changing this at all. If you are then,

If you are using the previously named properties for a version in your pom, you have to use [flatten-maven-plugin to do an `mvn install`
  or `mvn deploy`. More details can be found at [Maven CI Friendly Versions](/guides/mini/guide-maven-ci-friendly.html).

File: content/markdown/docs/3.5.0-beta-1/release-notes.md (hunk 1 above)
Comment (Bukama) [reply content/markdown/docs/3.5.0-beta-1/release-notes.md]: > I'm not clear on why you're changing this at all. If you are then,
> 
> If you are using the previously named properties for a version in your pom, you have to use [flatten-maven-plugin to do an `mvn install` or `mvn deploy`. More details can be found at [Maven CI Friendly Versions](/guides/mini/guide-maven-ci-friendly.html).

I'm only changing this to update the link in the release notes. It was not my intend to change the wording, but updated it to your suggestion.

**Review by elharo — APPROVED** [review]

File: content/markdown/guides/mini/guide-maven-ci-friendly.md
Comment (slawekjaranowski) [inline content/markdown/guides/mini/guide-maven-ci-friendly.md:1]: please also add redirect to .htaccess

Comment (Bukama) [reply content/markdown/guides/mini/guide-maven-ci-friendly.md:1]: Added

**Review by slawekjaranowski — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #1125

**Pull request merged**